offload/cmake/caches/AMDGPUBot.cmake

```diff
@@ -0,0 +1,13 @@
+set(LLVM_ENABLE_PROJECTS "clang;lld" CACHE STRING "")
+set(LLVM_ENABLE_RUNTIMES "compiler-rt;openmp;offload" CACHE STRING "")
+set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR ON CACHE BOOL "")
+set(LLVM_ENABLE_ASSERTIONS ON CACHE BOOL "")
+set(LLVM_LIT_ARGS "-v --show-unsupported --timeout 100 --show-xfail -j 32" CACHE STRING "")
+set(LLVM_TARGETS_TO_BUILD "X86;AMDGPU" CACHE STRING "")
```

Comment:
even though we don't support ARM yet, it still might be better to be a little bit future proof by using `host`.